protocol/requests/edit_message.go

```diff
@@ -4,16 +4,18 @@ import (
 	"errors"
 
 	"github.com/status-im/status-go/eth-node/types"
+	"github.com/status-im/status-go/protocol/common"
 	"github.com/status-im/status-go/protocol/protobuf"
 )
 
 var ErrEditMessageInvalidID = errors.New("edit-message: invalid id")
 var ErrEditMessageInvalidText = errors.New("edit-message: invalid text")
 
 type EditMessage struct {
-	ID          types.HexBytes                   `json:"id"`
-	Text        string                           `json:"text"`
-	ContentType protobuf.ChatMessage_ContentType `json:"content-type"`
+	ID           types.HexBytes                   `json:"id"`
+	Text         string                           `json:"text"`
+	ContentType  protobuf.ChatMessage_ContentType `json:"content-type"`
```

Comment:
I believe we should be using the more universal camel case convention for JSON keys and not tie status-go with Clojure's idioms.